compiler/circle-execution-plan/CMakeLists.txt

```diff
@@ -1,16 +1,3 @@
-set(SOURCES
```

Comment:
Maybe merge this and cmake file from src dir?

Reply:
Did I understand you correctly, that you are suggesting to use only this (`compiler/circle-execution-plan/CMakeLists.txt`) cmakeList and don't create new `compiler/circle-execution-plan/src/CMakeLists.txt`?